visualize/vis_utils.py

```diff
@@ -64,3 +64,28 @@ def save_npy(self, save_path):
             'length': self.real_num_frames,
         }
         np.save(save_path, data_dict)
+        
+        # Conversion of data for .pkl file for 'softcat477/SMPL-to-FBX/' 
+        # Change 'FbxTime.eFrames60's in SMPL-to-FBX/FbxReadWriter.py to 'FbxTime.eFrames30'
+        # Known issues: Glitches below 30 fps exports in 'softcat477/SMPL-to-FBX/' (20 fps expected)
+
```

Comment:
this seems to be missing the import statement?
`import utils.rotation_conversions as geometry`

Reply:
and 
`import pickle`

Reply:
Correct